next/schema-generator/definitions/zalozenie-komunitnej-zahrady.ts

```diff
@@ -0,0 +1,373 @@
+          object(
+            'mestoPsc',
+            { required: true },
+            {
+              columns: true,
+              columnsRatio: '3/1',
+            },
+            [
+              input('mesto', { title: 'Mesto', required: true }, {}),
+              input('psc', { title: 'PSČ', required: true, format: 'zip' }, {}),
+            ],
+          ),
+        ],
+      ),
+      input('email', { title: 'E-mail', required: true, type: 'email' }, {}),
+      input(
+        'telefon',
+        { title: 'Telefónne číslo', required: true, type: 'tel' },
+        { placeholder: '+421', size: 'medium' },
+      ),
+    ]),
+
+    step('obcianskeZdruzenie', { title: 'Občianske združenie' }, [
+      input(
+        'nazovObcianskehoZdruzenia',
+        { title: 'Názov občianskeho združenia', required: true, type: 'text' },
+        {},
+      ),
+      input('ico', { title: 'IČO', required: true, type: 'text' }, { size: 'medium' }),
+      object('adresaSidla', { required: true }, { objectDisplay: 'boxed', title: 'Adresa sídla' }, [
+        input('ulicaACislo', { title: 'Ulica a číslo', required: true, type: 'text' }, {}),
+        object(
+          'mestoPsc',
+          { required: true },
+          {
+            columns: true,
+            columnsRatio: '3/1',
+          },
+          [
+            input('mesto', { title: 'Mesto', required: true }, {}),
+            input('psc', { title: 'PSČ', required: true, format: 'zip' }, {}),
+          ],
+        ),
+      ]),
+      object(
+        'statutar',
+        { required: true },
+        {
+          title: 'Štatutár',
+          columns: true,
+          columnsRatio: '1/1',
+        },
+        [
+          input('menoStatutara', { title: 'Meno', required: true, type: 'text' }, {}),
+          input('priezviskoStatutara', { title: 'Priezvisko', required: true, type: 'text' }, {}),
+        ],
+      ),
+    ]),
+
+    step(
+      'pozemok',
+      {
+        title: 'Pozemok',
+        description:
+          'Odporúčame vám výber jedného z predschválených mestských pozemkov, vďaka čomu vieme žiadosť vybaviť rýchlejšie.',
+      },
+      [
+        radioGroup(
+          'typPozemku',
+          {
+            type: 'string',
+            title: 'O aký pozemok máte záujem?',
+            required: true,
+            options: [
+              { value: 'predschvaleny', title: 'Predschválený mestský pozemok', isDefault: true },
+              { value: 'iny', title: 'Iný mestský pozemok' },
+            ],
+          },
+          {
+            variant: 'boxed',
+            orientations: 'column',
+          },
+        ),
+        conditionalFields(createCondition([[['typPozemku'], { const: 'predschvaleny' }]]), [
+          select(
+            'predschvalenyPozemok',
+            {
+              title: 'Ponuka predschválených mestských pozemkov',
+              required: true,
+              options: [
+                { value: '1', title: 'Azalková' },
```

Comment:
Please also use `createStringOptions` for this, this is not stable long term, as "10" or any number can be easily changed for something else when the options are changed.